torchvision/prototype/models/depth/stereo/crestereo.py

```diff
@@ -1055,11 +1057,145 @@ def _crestereo(
     return model
 
 
+_COMMON_META = {
+    "min_size": (384, 512),
+}
+
+
 class CREStereo_Base_Weights(WeightsEnum):
-    pass
+    """The metrics reported here are as follows.
+
+    ``mae`` is the "mean-average-error" and indicates how far (in pixels) the
+    predicted disparity is from its true value (equivalent to ``epe``). This is averaged over all pixels
+    of all images. ``1px``, ``3px``, ``5px`` and indicate the percentage of pixels that have a lower
+    error than that of the ground truth. ``relepe`` is the "relative-end-point-error" and is the
+    average ``epe`` divided by the average ground truth disparity. ``f1`` corresponds to the average of pixels whose epe
+    is either >3px, or whom's ``relepe`` is higher than 0.05 (therefore lower is better).
+
+    """
+
+    MEGVII = Weights(
```

Comment:
Question, why the name is `MEGVII`?

Reply:
These are the weights uploaded by authors in their repo, with a training recipe that is unknown.

Reply:
Just realized it comes from the repo name: `megvii-research`. Just a little NIT, we should name it `MEGVII_V1` just in case the author release new weight and we want to name it `MEGVII_V2` later on.